powa/static/js/components/BreadCrumbs.vue

```diff
@@ -44,4 +51,8 @@ watch(
     });
   }
 );
+
+function onSelect(url) {
```

Comment:
I'm a bit afraid it's quite a generic name.  Maybe onBreadcrumbsSelect() or something like that?

Reply:
This function is **only** available in the component context.

Reply:
Oh then no objection